JS export template fails to compile

Hi, i was trying to compile javascript export templates but i am getting this error:

```
[ 57%] Compiling ==> core/node_path.cpp
thirdparty/misc/clipper.cpp:901:7: error: cannot use 'throw' with exceptions disabled
      throw clipperException("Coordinate outside allowed range");
      ^
thirdparty/misc/clipper.cpp:1049:5: error: cannot use 'throw' with exceptions disabled
    throw clipperException("AddPath: Open paths must be subject.");
    ^
thirdparty/misc/clipper.cpp:1081:5: error: cannot use 'throw' with exceptions disabled
    throw; //range test fails
    ^
thirdparty/misc/clipper.cpp:1065:3: error: cannot use 'try' with exceptions disabled
  try
  ^
thirdparty/misc/clipper.cpp:1445:5: error: cannot use 'throw' with exceptions disabled
    throw clipperException("UpdateEdgeIntoAEL: invalid call");
    ^
thirdparty/misc/clipper.cpp:1513:5: error: cannot use 'throw' with exceptions disabled
    throw clipperException("Error: PolyTree struct is needed for open path clipping.");
    ^
thirdparty/misc/clipper.cpp:1563:3: error: cannot use 'try' with exceptions disabled
  try {
  ^
thirdparty/misc/clipper.cpp:2841:5: error: cannot use 'throw' with exceptions disabled
    throw clipperException("ProcessIntersections error");
    ^
thirdparty/misc/clipper.cpp:2830:3: error: cannot use 'try' with exceptions disabled
  try {
  ^
thirdparty/misc/clipper.cpp:3005:8: error: cannot use 'throw' with exceptions disabled
  else throw clipperException("DoMaxima error");
       ^
10 errors generated.
ERROR:root:compiler frontend failed to generate LLVM bitcode, halting
scons: *** [thirdparty/misc/clipper.javascript.debug.bc] Error 1
scons: building terminated because of errors.
```
Command i used: ```scons platform=javascript -j4 target=debug```
Not sure if this is a already known error or i'm missing something :/
btw, are there nightly builds?

--- a/core/SCsub
+++ b/core/SCsub
@@ -68,7 +68,6 @@ thirdparty_sources = [
 	"md5.cpp",
 	"pcg.cpp",
 	"triangulator.cpp",
-	"clipper.cpp",
 ]
 thirdparty_sources = [thirdparty_dir + file for file in thirdparty_sources]
 env.add_source_files(env.core_sources, thirdparty_sources)

--- a/editor/SCsub
+++ b/editor/SCsub
@@ -466,6 +466,7 @@ if env['tools']:
 
 
     env.add_source_files(env.editor_sources, "*.cpp")
+    env.add_source_files(env.editor_sources, ["#thirdparty/misc/clipper.cpp"])
 
     SConscript('collada/SCsub')
     SConscript('doc/SCsub')

